src/main/java/com/salesforce/dynamodbv2/mt/mappers/MtAmazonDynamoDbBase.java

```diff
@@ -107,6 +109,12 @@
  */
 public class MtAmazonDynamoDbBase implements MtAmazonDynamoDb {
 
+    /**
+     * Special "column" key returned to client on multitenant scans.
+     */
+    public static final String TENANT_KEY = "mt:context";
+    public static final String VIRTUAL_TABLE_KEY = "mt:tableName";
```

Comment:
It would probably be good to make these attribute names configurable, so that clients can assign strings that they know for sure won't conflict with other physical or virtual attribute names.

Reply:
sounds good, will include that in a followup OR